packages/vue/src/DataDisplay/Table/OcTable.vue

```diff
@@ -220,119 +245,143 @@ onMounted(() => onScroll());
       </div>
     </template>
     <template v-else>
-      <slot
-        name="table-body"
-        :fields="fields"
-        :headers="headers"
-        :is-scroll-on-end="isScrollOnEnd"
-        :on-click-row="onClickRow"
-        :is-selectable="isSelectable"
-        :is-cursor-pointer="isCursorPointer"
-        :is-responsive="isResponsive"
-        :is-sticky="isSticky"
-        :row-link="rowLink"
-        :calculate-row-class="calculateRowClass"
+      <OcDraggable
+        :model-value="fields"
+        :is-disabled="!isDraggable"
+        @update:model-value="onDraggableUpdated($event)"
+        @detectMove="isRowMoving = true"
```

Comment:
can re-use this one `src/frontend/dashboard/resources/js/components/DraggableTable.vue`?
should not add more logic and make the component complicated

Reply:
I got it, thanks!